.github/workflows/release.yml

```diff
@@ -24,12 +24,10 @@ jobs:
     - run: make test-unit
     - run: make test-system
     - run: make test-heavy
-    - run: make create-release-assets
-    - name: upload release assets
-      uses: svenstaro/upload-release-action@1.1.0
+    - uses: goreleaser/goreleaser-action@v2
       with:
-        repo_token: ${{ secrets.GITHUB_TOKEN }}
-        tag: ${{ github.ref }}
-        file: dist/release/*
-        file_glob: true
-        overwrite: true
+        distribution: goreleaser
+        version: latest
+        args: release --rm-dist -f scripts/goreleaser.yaml
```

Comment:
Any reason not to put the config file in the standard location (`.goreleaser.yaml` in the root directory)?

Reply:
I just didn't want to add another file to the root when we're trying to cut down the number of files here, is this silly?

Reply:
I'll just put this in the standard location.